test/governance/AxelarServiceGovernance.js

```diff
@@ -179,24 +172,19 @@ describe('AxelarServiceGovernance', () => {
             .withArgs(proposalHash, target, calldata, nativeValue);
     });
 
-    it('should revert on executing a multisig proposal if called by non-signer', async () => {
+    it('should revert on executing a multisig proposal if called by non-multisig', async () => {
         const target = targetContract.address;
 
         await expect(
             serviceGovernance.connect(ownerWallet).executeMultisigProposal(target, calldata, 0),
-        ).to.be.revertedWithCustomError(serviceGovernance, 'NotSigner');
+        ).to.be.revertedWithCustomError(serviceGovernance, 'NotAuthorized');
```

Comment:
Can be done in a later PR but should use `expectRevert` from utils for live testnet compatibility